.github/workflows/global-ci.yaml

```diff
@@ -3,9 +3,30 @@ name: Global CI
 on: ["push", "pull_request"]
 
 jobs:
-  e2e:
+  e2e-main:
     uses: konveyor/ci/.github/workflows/global-ci.yml@main
+    if: "${{github.event.pull_request.base.ref == 'main' }}"
```

Comment:
To make it run for both `push` and `pull_request` - please use ` if: ${{ github.ref == 'main' || github.event.pull_request.base.ref == 'main' }}`
See [go-konveyor-tests workflow](https://github.com/konveyor/go-konveyor-tests/blob/main/.github/workflows/releases-tier0.yml#L16) for reference